src/main/java/io/github/cottonmc/resources/CottonResources.java

```diff
@@ -8,13 +8,12 @@
 public class CottonResources implements ModInitializer {
 
 	public static final String MODID = "cotton-resources";
-
-	public static ModLogger logger = new ModLogger(MODID, "COTTON RESOURCES");
-	public static CottonResourcesConfig config = ConfigManager.loadConfig(CottonResourcesConfig.class);
+	public static ModLogger LOGGER = new ModLogger(MODID, "COTTON RESOURCES");
```

Comment:
Wow, does this prefix our log entries with `[COTTON RESOURCES]`? Seems... loud

Reply:
It is a pun, right?